spec/features/guest_user_can_see_how_it_works_spec.rb

```diff
@@ -13,7 +13,7 @@
   it "clicks on about link" do
     visit '/'
 
-    click_on "About"
+    click_on "About", :first
```

Comment:
Okay so this should fix out "Ambiguous match" issue with rspec?

Reply:
Yeah, this fixes the ambiguous match.  I'm don't know why it's finding more than one, but I figure, for now, it's more important to get test passing so we can move on.

Reply:
Yeah I second that